### PR TITLE
framework:local-deploy command

### PR DESCRIPTION
**Pull request by xbill82**
⚠️ depends on #2 

Allows to create a local deployment of the current state of the framework. Repositories can be selected, so the process doesn't take too much time.

The frameworks and the repos are deployed on `/tmp/kuzzle-documentation`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +84,23 @@ OPTIONS
                                  (e.g. 6/) - env: $DOC_VERSION
 ```
 
-_See code: [src/commands/framework/link.ts](https://github.com/kuzzleio/kuzzle-documentation-cli/blob/v1.2.2/src/commands/framework/link.ts)_
+_See code: [src/commands/framework/link.ts](https://github.com/kuzzleio/kuzdoc/blob/v1.2.2/src/commands/framework/link.ts)_
+
+## `kuzdoc framework:local-deploy [FILE]`
+
+describe the command here
```
Comment (Njuelle) [inline README.md]: could we have a very little more detailed description? ;)

Comment (xbill82) [reply README.md]: DESCRIPTIONS ARE FOR THE WEAK

Comment (Aschen) [reply README.md]: READ DA CODE N00BZ

**Review by Njuelle — APPROVED** [review]

**Pull request merged**